universal-login-commons/lib/core/utils/securityCodes.ts

```diff
@@ -4,6 +4,7 @@ import {ensure} from './errors';
 import {isProperAddress, reverseHexString} from './hexStrings';
 import {shuffle, array8bitTo16bit} from './arrays';
 import {Notification} from '../models/notifications';
+const deepCopy = require('lodash.clonedeep');
```

Comment:
Don't use require, rename `deepCopy` to `clonedeep`, everywhere in the project. We don't code to reflect history.